run

```diff
@@ -119,6 +121,12 @@ function runstuff {
         && echo "DB setup with sample data"
     ;;
 
+    bootstrap:ui-static)
```

Comment:
`source/SIL.AppBuilder.Portal.Frontend/src/public/assets/language/ ` needs to be added to `.gitignore`

Reply:
it is in the frontend gitignore